scripts/common_startup.sh

```diff
@@ -109,15 +109,15 @@ if [ $SET_VENV -eq 1 ] && [ $CREATE_VENV -eq 1 ]; then
         # as well, but in this case we need it done beforehand.
         set_conda_exe
         if [ -n "$CONDA_EXE" ]; then
-            echo "Found Conda, will set up a virtualenv using conda."
+            echo "Found Conda, will set up a virtualenv using a python installed from conda."
             echo "To use a virtualenv instead, create one with a non-Conda Python at $GALAXY_VIRTUAL_ENV"
             : ${GALAXY_CONDA_ENV:="_galaxy_"}
             if [ "$CONDA_DEFAULT_ENV" != "$GALAXY_CONDA_ENV" ]; then
                 if ! check_conda_env "$GALAXY_CONDA_ENV"; then
                     echo "Creating Conda environment for Galaxy: $GALAXY_CONDA_ENV"
                     echo "To avoid this, use the --no-create-venv flag or set \$GALAXY_CONDA_ENV to an"
                     echo "existing environment before starting Galaxy."
-                    $CONDA_EXE create --yes --override-channels --channel conda-forge --name "$GALAXY_CONDA_ENV" "python=${MIN_PYTHON_VERSION}" "pip>=${MIN_PIP_VERSION}"
+                    $CONDA_EXE create --yes --override-channels --channel conda-forge --name "$GALAXY_CONDA_ENV" "python>=${MIN_PYTHON_VERSION}" "pip>=${MIN_PIP_VERSION}"
```

Comment:
This could install a Python version that we don't support yet (e.g. the upcoming 3.12), is it necessary?

Reply:
> The [docs](https://docs.galaxyproject.org/en/master/admin/python.html) currently just state that Python >=3.7 is supported. Maybe we want to be more specific there?

Yes, it would be good to specify also the latest Python version we are testing with.

> But I was also wondering if we should extend our testing here. As far as I get it we only test 3.7, or?

```shell
$ grep -r '3\.11' .github/workflows/
.github/workflows/test_galaxy_packages.yaml:        python-version: ['3.7', '3.11']
.github/workflows/first_startup.yaml:        python-version: ['3.7', '3.11']
.github/workflows/reports_startup.yaml:        python-version: ['3.7', '3.11']
.github/workflows/lint_openapi_schema.yml:        python-version: ['3.7', '3.11']
.github/workflows/osx_startup.yaml:        python-version: ['3.7', '3.11']
.github/workflows/lint.yaml:        python-version: ['3.7', '3.11']
.github/workflows/unit.yaml:        python-version: ['3.7', '3.11']
```

where `lint_openapi_schema.yml` and `osx_startup.yaml` where added in 23.1 .

Reply:
> I just followed the variable name here: MIN_PYTHON_VERSION. It's not necessary. Maybe we can reuse this:
https://github.com/galaxyproject/galaxy/blob/f2fcfa60f9e22f21179118e3f149ee01c02deb67/lib/galaxy/dependencies/update.sh#L10C41-L10C41?

This was already changed for 23.1 in commit 940ff9fe36af0a1b3ce02851fe2f8103b9dadabb so you can use the `GALAXY_CONDA_PYTHON_VERSION` environment variable to specify which Python version to use when creating the conda environment, would that work for you?

Reply:
Sounds reasonable. Thanks for all the helpful explanations.

Reply:
Cool, then I'll remove this change and add a small refactoring to this branch if that's OK?

Reply:
Sure.